toolkit/docs/building/building.md

```diff
@@ -486,9 +478,8 @@ To reproduce an ISO build, run the same make invocation as before, but set:
 | CONFIG_BASE_DIR               | `$(dir $(CONFIG_FILE)) `                                                                               | Base directory to search for image files in (see [image_config.md](../images/image_config.md))
 | TERMINAL_ISO_INSTALLER        | n                                                                                                      | Use a command line ISO installer instead of the GUI installer
 | UNATTENDED_INSTALLER          |                                                                                                        | Create unattended ISO installer if set. Overrides all other installer options.
-| PACKAGE_BUILD_LIST            |                                                                                                        | Additional packages to build
+| PACKAGE_BUILD_LIST            |                                                                                                        | Additional packages to build. The package may be skipped if the build system thinks it is already up-to-date.
```

Comment:
>may [](start = 181, length = 3)

Nit: let's use "will" instead of "may" so there's no doubt. Right now it feels like it also may not be skipped due to some unexplained reasons despite being up-to-date.